xc/xenops_server_xen.ml

```diff
@@ -919,6 +905,30 @@ module Featureset = struct
     |> String.concat ":"
 end
 
+let debug_featuresets xc name featureset featureset_max index_max =
+  debug "%s Max featureset = %a" name Featureset.pp featureset_max ;
+  debug "%s Dynamic set = %a" name Featureset.pp featureset ;
+
+  (* sanity checks: our featureset shouldn't be more than max! *)
+  let ours_minus_max = Featureset.diff featureset featureset_max in
+  if not @@ Array.for_all (fun x -> Int64.equal x 0L) ours_minus_max then (
+    debug "%s (DynamicSet - Max) = %a" name Featureset.pp ours_minus_max ;
+    warn
+      "Our default policy has CPUID features that are not present in Max \
+       policy! (see above)"
+  ) ;
+  debug "%s (Max-Default) = %a" name Featureset.pp
```

Comment:
You've got "DynamicSet - Max" above, so this ought to be "Max-Default" to match.